pkg/cri/server/container_create_windows.go

```diff
@@ -46,12 +47,16 @@ func (c *criService) containerSpec(
 	imageName string,
 	config *runtime.ContainerConfig,
 	sandboxConfig *runtime.PodSandboxConfig,
-	imageConfig *imagespec.ImageConfig,
+	image imagestore.Image,
 	extraMounts []*runtime.Mount,
 	ociRuntime config.Runtime,
 ) (*runtimespec.Spec, error) {
-	specOpts := []oci.SpecOpts{
-		customopts.WithProcessArgs(config, imageConfig),
+	imageConfig := &image.ImageSpec.Config
+	var specOpts []oci.SpecOpts
+	if image.ArgsEscaped {
```

Comment:
I see why you are doing this now. It really feels to me that you dont know this at this level and you should just call "merge" on it regardless. The "merge" code should determine to set Args vs CommandLine. IE: If you have both entrypoint and cmd from config, then the image ArgsEscaped doesnt matter at all. So we should handle it at the lower layer

Reply:
I have moved the check of argsEscaped to a lower level. Also removed a bunch of other unnecessary changes that were in the PR after OCI changes

Reply:
I have moved the check of argsEscaped to a lower level. Also removed a bunch of other unnecessary changes that were in the PR after OCI changes